src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,33 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
```

Comment:
Move that interface somewhere else and reuse it with Omit in MovieCard component